importer/api/__init__.py

```diff
@@ -142,6 +144,7 @@ def import_children(course, element, parent):
         content_xml=etree.tostring(element),
         mpath=mpath,
     )
+    target = "/static/"
```

Comment:
I don't have a problem with it either way. If you feel strongly about it I'll make that change.